apps/services/auth/admin-api/infra/auth-admin-api.ts

```diff
@@ -15,18 +15,6 @@ export const serviceSetup = (): ServiceBuilder<'services-auth-admin-api'> => {
         staging: 'https://identity-server.staging01.devland.is',
         prod: 'https://innskra.island.is',
       },
-      IDENTITY_SERVER_ISSUER_URL_LIST: {
```

Comment:
what happened?

Reply:
The `JwtStrategy` doesn't support multiple issuers as I thought. We need to provide new strategy per issuer to support this multi issuer setup, which requires a bit more thinking and implementation.